src/main/java/uk/gov/hmcts/cmc/claimstore/events/ccj/CountyCourtJudgmentEvent.java

```diff
@@ -13,12 +13,12 @@
 
     private final Claim claim;
     private final String authorisation;
-    private final boolean issue;
+    private final boolean isByAdmission;
```

Comment:
At this moment w.r.t to this story and its comparison with master, we've two types only. I would recommend doing this change in @dharmendrak story or other. However, CCJ by determination or refer to judge are sub categories under by admission only. This story is blocking other one in the play but it would be better we put this across.

Reply:
Please have a look at enum `CountyCourtJudgmentType`.

Reply:
This is already in master. The `Claim` object in this event contains the `CountyCourtJudgment` which has `ccjType` enum. we can use that.